content/docs/user-guide/experiment-management/cleaning-experiments.md

```diff
@@ -18,13 +18,28 @@ main:
     cnn-64
     cnn-128
 $ dvc exp remove cnn-32 cnn-64
-$ dvc exp list
-main:
-    cnn-128
+Removed experiments: cnn-32,cnn-64
 ```
 
 ## Removing multiple experiments
 
+You may wish to remove multiple experiments at once. For example, if you are
+finished experimenting on the current commit and want to discard all experiments
+based on that commit, use `--rev HEAD`.
+
+```
+$ dvc exp list
```

Comment:
Should we rename the title since it's a relatively narrow feature?

````suggestion
## Removing multiple experiments (based on parent commit)

You may wish to remove multiple experiments at once. For example, if you are
finished experimenting on the current Git commit and want to discard all
experiments derived from it, use `dvc exp remove --rev` with `HEAD`.

```dvc
$ dvc exp list
````